tools/db_crashtest.py

```diff
@@ -888,6 +869,23 @@ def finalize_and_sanitize(src_params):
         dest_params["metadata_write_fault_one_in"] = 0
         dest_params["read_fault_one_in"] = 0
         dest_params["metadata_read_fault_one_in"] = 0
+        if dest_params.get("txn_write_policy", 0) != 0:
+            # TODO: should any of this change for WUP (txn_write_policy==2)?
+            dest_params["wp_snapshot_cache_bits"] = 1
+            # try small wp_commit_cache_bits, e.g. 0 once we explore storing full
+            # commit sequence numbers in commit cache
+            dest_params["wp_commit_cache_bits"] = 10
+            # pipeline write is not currnetly compatible with WritePrepared txns
+            dest_params["enable_pipelined_write"] = 0
```

Comment:
redundant to https://github.com/facebook/rocksdb/pull/13499/files#diff-55ef501d94ad5940b8a033492dd9184d723a43a7676725c9bc638d25e13fed9dR641-R642 ?

Reply:
This (new 879) seems like a permanent disabling (no plan to fix) while the other seems more temporary (TODO). `enable_pipelined_write` handling is kind of a mess that I don't really understand and don't really want to sort out at the moment, so I'm just trying to preserve the past intentions, even if they're weird.